### PR TITLE
Update modal styles

### PR DESCRIPTION
**Pull request by LisaManresa**
- [x] Add the Clubhouse Story ID: [ch21482]

### CODE REVIEW

File: src/js/views/globals/modal/modal.hbs [hunk 1]
```diff
@@ -12,6 +12,6 @@
 <div class="modal-footer" data-footer-region>
   <div class="u-float--right">
     <a href="#" class="modal-cancel js-close">{{ cancelText }}</a>
-    <button type="submit" class="{{ buttonClass }} js-submit">{{ submitText }}</button>
+    <button type="submit" class="button--primary {{ buttonClass }} js-submit">{{ submitText }}</button>
```
Comment (paulfalgout) [inline src/js/views/globals/modal/modal.hbs]: I don't think `.button--primary` would go here.  If it's the better button than `button--blue` it should be the default, but `button--primary` is a specific style

File: src/sass/modules/modals.scss [hunk 2]
```diff
@@ -1,4 +1,6 @@
 .modal {
+  background-color: $white;
+  border-radius: 6px;
```
Comment (paulfalgout) [inline src/sass/modules/modals.scss]: we should maybe check with mark on this, but I think 6px is a copy over from mainapp.  this should likely be 4 or 8

File: src/sass/modules/modals.scss [hunk 3]
```diff
@@ -37,13 +36,7 @@
 }
 
 .modal-cancel {
```
Comment (paulfalgout) [inline src/sass/modules/modals.scss]: if this is all that's on this style I think we can use `u-margin--r-8`, but is there no hover/focus style at all for these cancels?

**Review by paulfalgout — CHANGES_REQUESTED** [review]

File: src/sass/modules/modals.scss (hunk 3 above)
Comment (LisaManresa) [reply src/sass/modules/modals.scss]: I'll check with Mark. Seems like there should be.

File: src/js/views/globals/modal/modal.hbs [hunk 4]
```diff
@@ -11,7 +11,7 @@
 
 <div class="modal-footer" data-footer-region>
   <div class="u-float--right">
-    <a href="#" class="modal-cancel js-close">{{ cancelText }}</a>
-    <button type="submit" class="{{ buttonClass }} js-submit">{{ submitText }}</button>
+    <button class="modal-cancel button--text js-close">{{ cancelText }}</button>
+    <button type="submit" class="button--blue {{ buttonClass }} js-submit">{{ submitText }}</button>
```
Comment (paulfalgout) [inline src/js/views/globals/modal/modal.hbs]: Ah so I mean that the value of `{{ buttonClass }}` should default to `blue--button` and no class should be added here.

**Review by paulfalgout — CHANGES_REQUESTED** [review]

File: src/js/views/globals/modal/modal.hbs [hunk 5]
```diff
@@ -11,7 +11,7 @@
 
 <div class="modal-footer" data-footer-region>
   <div class="u-float--right">
-    <a href="#" class="modal-cancel js-close">{{ cancelText }}</a>
-    <button type="submit" class="{{ buttonClass }} js-submit">{{ submitText }}</button>
+    <button class="modal-cancel button--text js-close">{{ cancelText }}</button>
```
Comment (paulfalgout) [inline src/js/views/globals/modal/modal.hbs]: :+1:

File: src/js/views/globals/modal/modal.hbs (hunk 4 above)
Comment (LisaManresa) [reply src/js/views/globals/modal/modal.hbs]: Ah, that makes way more sense.

File: src/js/views/globals/modal/modal.hbs (hunk 5 above)
Comment (paulfalgout) [reply src/js/views/globals/modal/modal.hbs]: we can actually remove `.modal-cancel` here now.  I assume it's not used anywhere

**Review by paulfalgout — APPROVED** [review]

**Comment by paulfalgout** [comment]
The CI failures here aren't related to this PR.  See: https://github.com/RoundingWell/care-ops-frontend/pull/159

**Pull request merged**